files/apt/sources.list

```diff
@@ -1,8 +1,14 @@
 ## Debian mirror on Microsoft Azure
 ## Ref: http://debian-archive.trafficmanager.net/
 
-deb http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
+deb-src [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
+deb-src [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+
+deb [arch=arm64,armhf] http://deb.debian.org/debian stretch main contrib non-free
+deb-src [arch=arm64,armhf] http://deb.debian.org/debian stretch main contrib non-free
+deb [arch=arm64,armhf] http://deb.debian.org/debian stretch-updates main contrib non-free
+deb-src [arch=arm64,armhf] http://deb.debian.org/debian stretch-updates main contrib non-free
+deb [arch=arm64,armhf] http://ftp.debian.org/debian stretch-backports main
```

Comment:
can you split this into three files? sources.list.amd64, sources.list.arm64, and sources.list.armhf?

Reply:
Done